grep/icon_grep/util/utils.py

```diff
@@ -1,58 +1,33 @@
-import komand
+from insightconnect_plugin_runtime.exceptions import PluginException
+from subprocess import run, CalledProcessError
```

Comment:
```suggestion
from subprocess import run
```
I don't see CalledProcessError used anywhere